Bottom: 1807660638e9678319a12f50ef6e6e5dc684ce66
Top:    ad285e7c2ce7efc944eea306cc528fc5fda857f1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 17:22:04 +0100

Refresh of action-c-fix-missing-prototype

---

diff --git a/src/action.c b/src/action.c
index 20dbb0a..faa1f21 100644
--- a/src/action.c
+++ b/src/action.c
@@ -55,6 +55,7 @@
 #include "line.h"
 #include "mymem.h"
 #include "misc.h"
+#include "mirror.h"
 #include "move.h"
 #include "polygon.h"
 /*#include "print.h"*/
diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index fb9b8cc..02448e5 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -840,14 +840,15 @@ ps_set_layer (const char *name, int group, int empty)
     fprintf (global.f, "gsave tx ty translate 1 -1 scale 0 0 moveto ( ) show grestore newpath /ty ty ts sub def\n");
 #endif
 
-  /* If we're printing a copper layer other than the outline layer,
-     and we want to "print outlines", and we have an outline layer,
+  /* If we're printing a layer other than the outline layer, and
+     we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
-  if (global.outline
-      && global.outline_layer != NULL
-      && global.outline_layer != PCB->Data->Layer+idx
-      && strcmp (name, "outline")
-      && strcmp (name, "route"))
+  if (global.outline &&
+      global.outline_layer != NULL &&
+      global.outline_layer != PCB->Data->Layer+idx &&
+      strcmp (name, "outline") != 0 &&
+      strcmp (name, "route") != 0
+      )
     {
       DrawLayer (global.outline_layer, &global.region);
     }
